### PR TITLE
Branded Welcome Template

### PR DESCRIPTION
**Pull request by coreybrown89**
This PR adds the Blitz branding and colors to the welcome template.

_*Note - not sure what the final documentation URL will be. This will probably be swapped out before the alpha launch._

Screen Shot 2020-04-20 at 3 25 57 PM

### CODE REVIEW

File: packages/cli/templates/app/app/pages/index.tsx [hunk 1]
```diff
@@ -115,15 +180,18 @@ const Home = () => (
...
     <style jsx global>{`
+      @import url('https://fonts.googleapis.com/css2?family=Libre+Franklin:wght@300;700&display=swap');
+
       html,
       body {
         padding: 0;
         margin: 0;
-        font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen, Ubuntu, Cantarell, Fira Sans,
-          Droid Sans, Helvetica Neue, sans-serif;
+        font-family: 'Libre Franklin', sans-serif;
```
Comment (Zeko369) [inline packages/cli/templates/app/app/pages/index.tsx:189]: Add back default fallbacks for people who are doing this offline

File: packages/cli/templates/app/app/pages/index.tsx [hunk 2]
```diff
@@ -8,9 +8,34 @@ const Home = () => (
...
 
     <main>
+      <div className="logo">
+        <img src="/logo.png" alt="blitz icon" height="72" width="72" />
+      </div>
       <h1 className="title" style={{marginBottom: 24}}>
         Welcome to Blitz.js
       </h1>
+      <p>
+        Get started by editing this page.{' '}
```
Comment (Zeko369) [inline packages/cli/templates/app/app/pages/index.tsx:18]: The code part is in the new line so this space is not required

File: packages/cli/templates/app/app/pages/index.tsx [hunk 3]
```diff
@@ -8,9 +8,34 @@ const Home = () => (
     </Head>
 
     <main>
+      <div className="logo">
+        <img src="/logo.png" alt="blitz icon" height="72" width="72" />
```
Comment (Zeko369) [inline packages/cli/templates/app/app/pages/index.tsx:12]: Isn't this height/width redundant having the same styles in css already?

File: packages/cli/templates/app/app/pages/index.tsx (hunk 1 above)
Comment (flybayer) [reply packages/cli/templates/app/app/pages/index.tsx:189]: Yeah, keep what was there but add 'Libre Franklin' to the beginning of the list for folks who have it installed.

File: packages/cli/templates/app/app/pages/index.tsx [hunk 4]
```diff
@@ -8,9 +8,34 @@ const Home = () => (
...
+      <p>
+        Get started by editing this page.{' '}
+        <pre>
+          <code>/app/pages/index.tsx</code>
+        </pre>
+      </p>
+      <div className="buttons">
+        <a
+          className="button"
+          href="https://blitzjs.com?utm_source=blitz-new&utm_medium=app-template&utm_campaign=blitz-new"
```
Comment (flybayer) [inline packages/cli/templates/app/app/pages/index.tsx:26]: Link to `https://github.com/blitz-js/blitz/blob/master/USER_GUIDE.md` for now

Comment (flybayer) [reply packages/cli/templates/app/app/pages/index.tsx:26]: Fixed in my branch

File: packages/cli/templates/app/app/pages/index.tsx (hunk 2 above)
Comment (flybayer) [reply packages/cli/templates/app/app/pages/index.tsx:18]: fixed in my branch

File: packages/cli/templates/app/app/pages/index.tsx (hunk 3 above)
Comment (flybayer) [reply packages/cli/templates/app/app/pages/index.tsx:12]: fixed in my branch

File: packages/cli/templates/app/app/pages/index.tsx (hunk 1 above)
Comment (flybayer) [reply packages/cli/templates/app/app/pages/index.tsx:189]: fixed in my branch

**Comment by flybayer** [comment]
Thank you for this Corey!!

I made some styling updates and added the full logo. My changes are on the `bb-template` branch in this repo. So pull that and merge into your branch.

**Comment by coreybrown89** [comment]
Done and ready for re-review.

File: packages/cli/templates/app/app/pages/index.tsx [hunk 5]
```diff
@@ -3,27 +3,22 @@ import {Head} from '@blitzjs/core'
 const Home = () => (
   <div className="container">
     <Head>
-      <title><%= name %></title>
+      <title>yep</title>
```
Comment (flybayer) [inline packages/cli/templates/app/app/pages/index.tsx]: Oops, this was my bad

```suggestion
      <title><%= name %></title>
```

**Pull request merged**